proto/path/protocol/morse.proto

```diff
@@ -3,16 +3,62 @@ package path.protocol;
 
 option go_package = "github.com/buildwithgrove/path/observation/protocol";
 
+// MorseEndpointErrorType enumerates possible relay errors when interacting with Morse endpoints
+enum MorseEndpointErrorType {
+  MORSE_ENDPOINT_ERROR_UNSPECIFIED = 0;
+  MORSE_ENDPOINT_ERROR_CONNECTION_FAILED = 1;
+  MORSE_ENDPOINT_ERROR_TIMEOUT = 2;
+  MORSE_ENDPOINT_ERROR_MAXED_OUT = 3;
+  MORSE_ENDPOINT_ERROR_MISCONFIGURED = 4;
+  MORSE_ENDPOINT_ERROR_INVALID_RESPONSE = 5;
+  MORSE_ENDPOINT_ERROR_VALIDATION_FAILED = 6;
+  MORSE_ENDPOINT_ERROR_INTERNAL = 7;  // Added for internal errors
+}
+
+// MorseSanctionType specifies the duration type for endpoint sanctions
+enum MorseSanctionType {
+  MORSE_SANCTION_UNSPECIFIED = 0;
+  MORSE_SANCTION_SESSION = 1;  // Valid only for current session
```

Comment:
Not sure if this is the right place for it but I think some more detailed comments on the types of sanctions would be useful for PATH users.